src-files/def-config.yaml

```diff
@@ -13,6 +13,9 @@ traversal:
     # Stack files
     - .stack-work
 
+    # Test files
+    - tests/markdowns
+
```

Comment:
Btw, this should not be here as this config is what `xrefcheck dump-config` produces. Consider updating this field from code if necessary.

Reply:
Oh
AFAIU we need `tests/markdowns` folder to be at `ignored` when we execute `xrefcheck` on itself, especially on `xrefcheck`'s root or `tests` (execution on another directory of `xrefcheck` must work fine). But how can we define it properly? Can we just check if the root's last directories are `xrefcheck` or `xrefcheck/tests`?

Reply:
[Updating from code](https://github.com/serokell/xrefcheck/pull/53/commits/13e551dc48ce602082f778ded55781aed4fb47ed) ✔️

Reply:
While this assumption that we appear in `xrefcheck/test` folder only in tests seems plausible, I think that `Main` on itself should not know anything about test-related stuff, because if we start doing so for other similar demands of test code, then the production logic will eventually become a sort of mess.

Normally the necessary behavior should be configurable and updated respectively in test logic. In our case this can look as follows:
* Extract a piece of logic from `defaultAction`, so that this piece accepts `Config` and performs the necessary checks.
* Invoke this method in tests, passing a config constructed manually.

Reply:
We discussed it separately that `tests/markdowns` should be ignored through github CLI options, not from `Main`